pages/index.js

```diff
@@ -18,77 +16,15 @@ const Home = () => {
   const router = useRouter();
   const auth = getAuth(firebaseApp);
 
-  const [loginEmail, setLoginEmail] = useState('');
-  const [loginPassword, setLoginPassword] = useState('');
-  const [signupEmail, setSignupEmail] = useState('');
-  const [signupPassword, setSignupPassword] = useState('');
-  const [signupPassConf, setSignupPassConf] = useState('');
-  const [loginNotification, setLoginNotification] = useState('');
-  const [signupNotification, setSignupNotification] = useState('');
-  const [logoutNotification, setLogoutNotification] = useState('');
 
-
-  const handleLogin = (e) => {
-    e.preventDefault();
-    signInWithEmailAndPassword(auth, loginEmail, loginPassword)
-      .then(
-        setLoginNotification(
-          'You are logged in'
-        ),
-        setTimeout(() => {
-          setLoginNotification('')
-        }, 2000)
-      )
-      .catch((err) => {
-        console.log(err.code, err.message)
-        setLoginNotification(err.message)
-        setTimeout(() => {
-          setLoginNotification('')
-        }, 2000)
-      })
-    setLoginEmail('')
-    setLoginPassword('')
-  }
-
-  const handleSignup = (e) => {
-    e.preventDefault();
-    if (signupPassword !== signupPassConf) {
-      setSignupNotification(
-        'Password and password confirmation does not match'
-      )
-      setTimeout(() => {
-        setSignupNotification('')
-      }, 2000)
-      setSignupPassword('');
-      setSignupPassConf('');
-      return null;
-    }
-    createUserWithEmailAndPassword(auth, signupEmail, signupPassword)
-      .catch((err) => {
-        console.log(err.code, err.message)
-      });
-  }
-
-  const handleLogout = (e) => {
-    e.preventDefault();
-    signOut(auth)
-      .then(() => {
-        setLogoutNotification('You are logged out')
-        setTimeout(() => {
-          setLogoutNotification('')
-        }, 2000),
-          console.log('User is logged out')
-      });
-  }
-
-  onAuthStateChanged(auth, (user) => {
-    if (user) {
-      const uid = user.uid;
-      console.log(uid);
-    } else {
-      console.log('User is signed out');
-    }
-  });
+  // onAuthStateChanged(auth, (user) => {
```

Comment:
Does this mean that the `onAuthStateChanged()` monitor is unnecessary here in this Home component/page? Then why do you import the module up on top of the file? What differences does it make if all it does,from what I see, is to log the user ID when signed in and give a message when siged out? Does it do other things behind the scence?

Reply:
Agreed. I think it is related to the component where the user lists are being fetched (Sidebar/TodoListsNav?)